chart/templates/rbac/pod-cleanup-role.yaml

```diff
@@ -18,7 +18,7 @@
 ################################
 ## Airflow Cleanup Role
 #################################
-{{- if and .Values.rbacEnabled .Values.cleanup.enabled }}
+{{- if and .Values.rbac.create .Values.cleanup.enabled }}
```

Comment:
Why `rbac.create` here, but `cleanup.serviceAccount.create` elsewhere?

More specifically: why do we need _two_ different config options for each  role we create?

Reply:
With the [new way](https://github.com/apache/airflow/pull/14152/files#diff-dc12e5cc8016c85ad9e662af7c8b8cfb49d91965225e94aa12146dc1f371fc86R472-R480) that this service account is generated, a user can technically set `rbac.create` to `true`, `.Values.cleanup.serviceAccont.create` to `false` and still use the namespaces [default](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#use-multiple-service-accounts) service account. In that instance, the above role doesn't necessarily care if a service account for `cleanup` has been created as its going to use the `default` service account instead.